express/parsers/molecule.py

```diff
@@ -9,60 +10,70 @@
 import logging
 import tempfile
 
+
 class MoleculeParser():
     """
     Molecule parser class.
 
     Args:
-        args (list): args passed to the parser.
-        kwargs (dict): kwargs passed to the parser.
-            structure_string (str): structure string.
-            structure_format (str): structure format, poscar or espresso-in.
+        structure_string (str): structure string.
+        structure_format (str): structure format, poscar or espresso-in.
     """
 
-    def __init__(self, structure_string):
+    def __init__(self, structure_string: str, structure_format: str):
         self.structure_string = structure_string
+
+        if structure_format == "poscar":
+            self.ase_format = "vasp"
+        else:
+            self.ase_format = structure_format
```

Comment:
The logic here is a bit ambiguous - what are we actually expecting as structure_format? Should we define a separate function (or just a lookup table, JSON dictionary) that converts between our structure format to ase?

Reply:
I believe we get one of three options for `structure_format` in our code: `poscar` or `espresso-in` or `cif`.
in ase `poscar` = `vasp`, `espresso-in` = `espresso-in` and `cif` = `cif`. So we only need to convert the name of the structure format for `poscar` the others don't need to be changed for use with ase.

If we are going to be adding a wide variety of file formats then using some kind of lookup table might be easier to handle because each type would just need a new entry in the table. But if we are planning to keep file types limited then a function might be sufficient.